scripts/pi-hole/php/header.php

```diff
@@ -343,7 +343,7 @@ function pidofFTL()
                                 {
                                     echo "text-vivid-blue";
                                 }
-                                ?>"\"></i> Temp:&nbsp;<span id="rawtemp" hidden><?php echo $celsius;?></span><span id="tempdisplay"></span><?php
+                                ?>"></i> Temp:&nbsp;</span><span id="rawtemp" hidden><?php echo $celsius;?></span><span id="tempdisplay"></span><?php
```

Comment:
I'm glad to see someone already had a PR up for this!  I was just about to put one up myself.
Though I think the new span closing tag is in the wrong place.

```suggestion
                                ?>"></i> Temp:&nbsp;<span id="rawtemp" hidden><?php echo $celsius;?></span><span id="tempdisplay"></span></span><?php
```